modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -567,22 +624,46 @@ RenderablePointCloud::RenderablePointCloud(const ghoul::Dictionary& dictionary)
         _unit = DistanceUnit::Meter;
     }
 
-    _spriteTexturePath.onChange([this]() { _spriteTextureIsDirty = true; });
-    addProperty(_spriteTexturePath);
-
-    _useSpriteTexture = p.useTexture.value_or(_useSpriteTexture);
-    addProperty(_useSpriteTexture);
+    addPropertySubOwner(_texture);
 
     if (p.texture.has_value()) {
-        _spriteTexturePath = absPath(*p.texture).string();
+        const Parameters::Texture t = *p.texture;
+
+        // Read texture information. Multi-texture is prioritized over single-texture
+        if (t.folder.has_value()) {
+            _textureMode = TextureInputMode::Multi;
+            _hasSpriteTexture = true;
+            _texturesDirectory = absPath(*t.folder).string();
+
+            if (t.file.has_value()) {
+                LWARNING(fmt::format(
+                    "Both a single texture File and multi-texture Folder was provided. "
+                    "The folder '{}' has priority and the single texture with the "
```

Comment:
This is also related to the discussion about whether std::filesystem::path automatically adds quotation marks